nym-vpn-app/src/screens/node/list/GatewayItem.tsx

```diff
@@ -98,19 +98,19 @@ const GatewayItem = ({
       {streamOptimized && (
         <MsIcon icon="smart_display" className="text-cornflower" />
       )}
-      <div className="flex py-2 self-stretch items-center">
+      <div className="flex p-2 self-stretch items-center">
         <Button
           className={clsx(
-            'w-14 flex justify-center items-center mr-3 shrink-0',
-            'text-baltic-sea/80 dark:text-white/80',
-            'hover:text-baltic-sea dark:hover:text-white',
+            'w-11 h-11 flex justify-center items-center shrink-0 rounded-full',
```

Comment:
w-11 sounds small compared to the previous 14 value, maybe an in-between size would be 12?